src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/runtime:6.0 AS base
+WORKDIR /app
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src
+COPY ["nuget.config", "."]
+COPY ["src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.csproj", "src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/"]
+COPY ["src/console/Microsoft.Health.Fhir.Ingest.Console.Common/Microsoft.Health.Fhir.Ingest.Console.Common.csproj", "src/console/Microsoft.Health.Fhir.Ingest.Console.Common/"]
+COPY ["src/lib", "src/lib"]
+RUN dotnet restore "src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.csproj"
+COPY . .
+WORKDIR "/src/src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation"
```

Comment:
Is this line correct? It begins with `/src/src/`

Reply:
Yes, the line is correct because of the WORKDIR /src on line 7, but it is a good callout that the double src is redundant. I will remove line 7 for clarity.